plugins/scanners/scanoss/src/main/kotlin/ScanOssConfig.kt

```diff
@@ -26,7 +26,8 @@ data class ScanOssConfig(
     @OrtPluginOption(defaultValue = "https://api.osskb.org/")
     val apiUrl: String,
 
-    /** The API key required to authenticate with the ScanOSS server. */
+    /** The API key used to authenticate with the ScanOSS server. */
+    @OrtPluginOption(defaultValue = "")
```

Comment:
Another option would be to make the property nullable, this would a bit "cleaner" on the config side but require some `orEmpty()`  call somewhere else in the code.

Reply:
I'd like to be a bit careful here and stick more closely to the previous semantics.

Reply:
Looks like making the `apiKey` nullable [is fine](https://github.com/oss-review-toolkit/ort/pull/10004/files#diff-ee996bc1eddf3620a51c2b96986a038a9718fba22893ba6f93823ecadfc79617R30) after all.